### ISSUE TITLE
member settings page (default avatar) page error

### ISSUE THREAD

**Issue by maloja** [oveleon/contao-member-extension-bundle#22]
Member extension with contao 5.3 shows up a page error "Internal Server Error, Class File not found" as soon as i try to configure the default avatar (Members ->Settings).

**Comment by zoglo** [comment]
@maloja can you be more specific? I can not reproduce this issue

- Please use the debug mode and post the stacktrace
- What settings are you talking about? The Backend settings?
- Are you running the latest version?

**Comment by maloja** [comment]
Sorry for being a bit unspecific, but here are the details. The error happens in the backend as soon as I click on settings in the members. (where you can set the default profile picture). Then only an error page appears.

Environment:  Contao: 5.3.8 with  contao-member-extension-bundle 1.4.1 / php8.2

Stack Trace:
```
ClassNotFoundError

Symfony\Component\ErrorHandler\Error\ClassNotFoundError:
Attempted to load class "File" from the global namespace.
Did you forget a "use" statement for e.g. "SimplePie\Cache\File", "SimplePie\File", "webignition\RobotsTxt\File\File", "ZipStream\File", "Symfony\Component\Mime\Part\File", "Symfony\Component\HttpFoundation\File\File", "PhpParser\Node\Scalar\MagicConst\File", "PhpOffice\PhpSpreadsheet\Shared\OLE\PPS\File" or "PhpOffice\PhpSpreadsheet\Shared\File"?

  at vendor/contao/core-bundle/contao/classes/Backend.php:291
  at Contao\Backend->getBackendModule('member_settings', null)
     (vendor/contao/core-bundle/contao/controllers/BackendMain.php:144)
  at Contao\BackendMain->run()
     (vendor/contao/core-bundle/src/Controller/BackendController.php:44)
  at Contao\CoreBundle\Controller\BackendController->mainAction()
     (vendor/symfony/http-kernel/HttpKernel.php:181)
  at Symfony\Component\HttpKernel\HttpKernel->handleRaw(object(Request), 1)
     (vendor/symfony/http-kernel/HttpKernel.php:76)
  at Symfony\Component\HttpKernel\HttpKernel->handle(object(Request), 1, true)
     (vendor/symfony/http-kernel/Kernel.php:197)
  at Symfony\Component\HttpKernel\Kernel->handle(object(Request))
     (public/index.php:42)                
```

let me know if you don't need additional information

**Comment by zoglo** [comment]
@maloja yeah, that made it more clear 😄 

Fixed in 1.4.2:
https://github.com/oveleon/contao-member-extension-bundle/releases/tag/1.4.2

**Comment by maloja** [comment]
@zoglo thanks for your super fast fix. Works perfect.
